packages/terra-profile-image/tests/nightwatch/ProfileImageDefault.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Image from 'terra-image';
+
+import ProfileImage from '../../lib/ProfileImage';
+import placeholder150x150 from '../assets/150x150.jpg';
```

Comment:
Do we really want to have 150x150 in the variable name? As we have size modifiers like: tiny, small, medium, large, huge, could we rename the image file to `mediumPlaceHolder.jpg`?